usecases/backup/scheduler.go

```diff
@@ -324,10 +322,8 @@ func (s *Scheduler) List(ctx context.Context, principal *models.Principal, backe
 	defer func(begin time.Time) {
 		logOperation(s.logger, "list_backup", "", backend, time.Now(), err)
 	}(time.Now())
-	if err := s.authorizer.Authorize(principal, authorization.READ, authorization.Backups(backend)); err != nil {
-		return nil, err
-	}
```

Comment:
**Suggestion:** if keeping the above commented-out code, should we keep this here but commented-out? If removing above then ignore this one!

Reply:
i think i will remove all the commented code , good catch